docs/09_firebase_deployment.md

```diff
@@ -0,0 +1,187 @@
+
+### Building and deploying Flutter application
+ 
+1. Open the terminal and navigate to the metrics project folder.
+2. Run `firebase login` command and follow the instructions to log in to the Firebase CLI with your Google account.
+ Use the same account that you used to create your Firebase project (or the one that has access to it).
+3. After you have logged in, run the `firebase use --add` command, and select the project id of the project, created in previous steps.
+4. Give an alias to your project.
+5. Run the `flutter build web` command from the root of the metrics project to builds the release version of the application.
+ Recommended that you add `--dart-define=FLUTTER_WEB_USE_SKIA=true` parameter to build the application with the `SKIA` renderer.
+4. Run the `firebase deploy --only hosting` command to deploy an application to the Firebase Hosting.
+
+After the deployment process finished, your application will be accessible using the `Hosting URL`, printed to console.
+
+## Creating test data for deployed application
+
+Once you've deployed the metrics application to the Firebase Hosting, you can create a sample data to test the application.  
+The application reads data from 2 firestore collections: `projects` and `build`. To create a sample data follow the next steps: 
+
+1. Deploying seedData Firestore function: 
+    1. Go to the `metrics/firebase/index.js` file and comment this line of code: `return resp.status(200).send('done');`.
+    2. Open console, navigate to `metrics` directory and run `firebase deploy --only functions`
+     command to deploy this cloud function.
+    3. Once the function deployment finished, the URL will be printed to the console.
+     You can trigger this function to create builds from the project, using this URL.  
+     This HTTP function has such query parameters: 
+        - buildsCount (required) - number of builds to be generated.
+        - projectId (required) - the project identifier for which builds will be generated.
+        - startDate (optional) - builds will be generated with 'startedAt'
+         property in the range from startDate to startDate - 7 days. Defaults to the current date.
+        - delay (optional) - is the delay in milliseconds between adding builds to the project.
+
+2. Creating projects in Firestore Database:
+    1. Go to [Firebase Console](https://console.firebase.google.com/), and select the project, created in previous steps.
+    2. Go to the database section on the left panel and tap on the `Start collection` button.
+    3. Add collection with `projects` identifier and tap `Next`. 
+    4. In the document creation window tap on the `Auto-ID` button or enter the project identifier you want.
+    5. Add a field named `name` with the `String` type and the name for your project as a value.
+
+# Dependencies
+
+> What is the project blocked on?
+
+This project has no dependencies.
+
+> What will be impacted by the project?
+
+Nothing will be impacted by this project.
```

Comment:
I think this project does not impact anything because we are just describing the process of deployment to Firebase Hosting but not changing in. @illia-romanenko, what do you think about this?

Reply:
I think @vladlaba description makes sense. So I would probably suggest to use it.